training/finetune_RedPajama-INCITE-Chat-3B-v1.sh

```diff
@@ -0,0 +1,56 @@
+DIR=$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd)
+
+netif=lo
+export GLOO_SOCKET_IFNAME=${netif}
+export NCCL_SOCKET_IFNAME=${netif}
+export MODEL_NAME=rp-incite-chat-3b-fintuned
```

Comment:
One last comment. Can we call this "redpajama-incite-chat-3b-sample" or "-demo" since this doesn't reproduce the 3B chat model that was released?